dhcp/dhcp/dhcpd_update/alcatel.conf

```diff
@@ -10,6 +10,9 @@ class "Alcatel-NOE" {
 
     # 'next-server' is used when the phone is in NOE mode
     next-server = config-option VOIP.tftp-server-name;
+    # 'filename' is used when specific phone is in NOE mode
+    filename = “sipconfig.txt”
```

Comment:
Are you on Mac now? Your `"` are `“` (do you see the difference?)

Reply:
Yes I see ! very strange... it's corrected